plugins/modules/storage/netapp/na_cdot_aggregate.py

```diff
@@ -18,7 +18,7 @@
 
 short_description: Manage NetApp cDOT aggregates.
 extends_documentation_fragment:
-- community.general.netapp.ontap
+- community.general._netapp.ontap
 
```

Comment:
This module is deprecated.  It is actually _na_cdot_aggregate.py.


I would rather have it deleted, as it is introducing confusion.

Reply:
The `_` mechanism for modules is no longer supported in collections. Instead, metadata (and once ansible/ansible#67684 has been merged, metadata parsed by that) is used.

The module has been marked as deprecated with a `removed_in_version` of 2.11, i.e. it should stay until Ansible 2.11 is released.